packages/jsp-ws-connection/README.md

```diff
@@ -0,0 +1,3 @@
+# new-lib
+
+TODO fill in the description
```

Comment:
@shoopapa - This all looks great!  The only thing is that since this will end up setting up a package, we will want to have a relatively informative README.md so people who use it know how to use it